README.md

```diff
@@ -1 +1,77 @@
-# portfolio_mobile_day3
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: setup and mobile first
+
+> Description the project.
+
+
+Set up a toolbar and headline section (right after the header):
+
+In order to lay out the elements on the page, Flexbox was used in all 2 sections.
+
+Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+Added descriptive README file my project.
+
```

Comment:
- I think these lines are not necessary for your readme file, You can as well take them off.
- Please, I'd like to kindly suggest that you add a description to your readme as well. 🥂

Reply:
Hi Favour, what do you mean by description to README. I already added description to my project on README as you can see they were separated by lines. I have removed the lines. Let me know in details so i can effect correctly.

Reply:
Hi @EmekaEmmanuel, What @Favourezeugwa meant was that the section after your project title in your readme should describe what your project is all about.
![read](https://user-images.githubusercontent.com/94551484/198511503-90ea6177-5806-44c9-a75a-7f46a3e40696.png)